packages/storefront-events-collector/src/handlers/requisitionList/removeFromRequisitionListAEP.ts

```diff
@@ -0,0 +1,49 @@
+import { Event } from "@adobe/magento-storefront-events-sdk/dist/types/types/events";
+
+import { sendEvent } from "../../alloy";
+import { BeaconSchema } from "../../types/aep";
+
+const XDM_EVENT_TYPE = "commerce.requisitionListRemovals";
+
+/** Sends an event to aep with an addToCart payload */
+const aepHandler = async (event: Event): Promise<void> => {
+    const { requisitionListItemsContext, debugContext, customContext, requisitionListContext, storefrontInstanceContext } = event.eventInfo;
+
+    let payload: BeaconSchema;
+    if (customContext && Object.keys(customContext as BeaconSchema).length !== 0) {
+        // override payload on custom context
+        payload = customContext as BeaconSchema;
+    } else {
+        payload = {
+            commerce: {
+                requisitionList: {
+                    ID: requisitionListContext?.id,
+                    name: requisitionListContext?.name,
+                    description: requisitionListContext?.description
+                },
+            },
+            productListItems: requisitionListItemsContext?.items?.map((item) => {
```

Comment:
Should we map this to a function of it's own? This logic is going to be reused quite a bit. We can refactor in another PR.